Tools/Create-Azure-Sentinel-Solution/input/Solution_DigitalShadowsSearchlight.json

```diff
@@ -0,0 +1,15 @@
+ {
+    "Name": "DigitalShadowsSearchlight",
```

Comment:
Is this the human-readable name shown in the listing? If so can we change this to "Digital Shadows SearchLight" please